### ISSUE TITLE
error on space between operators

### ISSUE THREAD

**Issue by ebess** [sasstools/sass-lint#483]
The following code:
```sass
$foo: 15px;

.foo {
    margin-top: - ($foo * 2);
}
```

Produces the following error:
![selection_062](https://cloud.githubusercontent.com/assets/5565003/12584683/f9d1b134-c448-11e5-8388-4085bb0c4da7.png)

**Comment by DanPurdy** [comment]
This would result in invalid css anyway? I agree it shouldn't throw an error though.

**Comment by ebess** [comment]
I don't know exactly, but if so, then the space between operators check should ignore such cases:
```sass
```
$foo: 15px;

.foo {
    margin: -($foo * 2);
}
```
warning  Space expected around operator  space-around-operator
```

**Comment by DanPurdy** [comment]
Ah, I see the real problem now! Thanks for the report :+1:
